src/OmniSharp.Roslyn.CSharp/Services/RoslynFeaturesHostServicesProvider.cs

```diff
@@ -12,7 +14,7 @@ public class RoslynFeaturesHostServicesProvider : IHostServicesProvider
         public ImmutableArray<Assembly> Assemblies { get; }
 
         [ImportingConstructor]
-        public RoslynFeaturesHostServicesProvider(IAssemblyLoader loader)
+        public RoslynFeaturesHostServicesProvider(IAssemblyLoader loader, OmniSharpOptions options, IOmniSharpEnvironment env)
```

Comment:
Also, I just realized that `RoslynFeaturesHostServicesProvider` is not marked as `[Shared]`. It probably doesn't matter in practice (since `OmniSharpWorkspace` is a singleton), but I just noticed.